tower/Cargo.toml

```diff
@@ -91,7 +91,6 @@ tower-test = { version = "0.4", path = "../tower-test" }
 tracing = { version = "0.1.2", default-features = false, features = ["std"] }
 tracing-subscriber = { version = "0.3", default-features = false, features = ["fmt", "ansi"] }
 http = "0.2"
-lazy_static = "1.4.0"
```

Comment:
I think this is needed to fix the minimal version used in tests, per #670, or else you get:

```
error: cannot find macro `__lazy_static_internal` in this scope
  --> [...]/sharded-slab-0.1.1/src/tid.rs:32:1
   |
32 | / lazy_static! {
33 | |     static ref REGISTRY: Registry = Registry {
34 | |         next: AtomicUsize::new(0),
35 | |         free: Mutex::new(VecDeque::new()),
36 | |     };
37 | | }
   | |_^
   |
   = note: this error originates in the macro `lazy_static` (in Nightly builds, run with -Z macro-backtrace for more info)
```